server/src/main/java/org/apache/uniffle/server/storage/MultiStorageManager.java

```diff
@@ -71,7 +71,7 @@ public void updateWriteMetrics(ShuffleDataFlushEvent event, long writeTime) {
   @Override
   public boolean write(Storage storage, ShuffleWriteHandler handler, ShuffleDataFlushEvent event) {
     StorageManager storageManager = selectStorageManager(event);
-    if (storageManager == coldStorageManager && event.getRetryTimes() > fallBackTimes) {
+    if (event.getRetryTimes() > fallBackTimes) {
```

Comment:
Why not put these select condition into one abstract method? such as being involved in `selectStorageManager`